ruby/exercises/promedio_array.rb

```diff
@@ -11,3 +11,19 @@
 #     Input: [1, 5, 87, 45, 8, 8] Resultado: 25
 #     Input: [2,5,13,20,16,16,10] Resultado: 11
 #     Input: [1,2,15,15,17,11,12,17,17,14,13,15,6,11,8,7] Resultado: 11
+def average(values)
+    unless values.empty?
+    p "el promedio de #{values} es:"
+    values.sum / values.size
+    end
```

Comment:
Podrias usar un guard clause para no crear un `unless ... end`
```suggestion
    return if values.empty?
    
    p "el promedio de #{values} es:"
    values.sum / values.size
```